app/serializers/debate_serializer.rb

```diff
@@ -1,10 +1,15 @@
 class DebateSerializer
-  def initialize(debate)
+  def initialize(debate, auth_token)
```

Comment:
you dont need to pass the whole auth token object here. i think you only need auth_token.id